### ISSUE TITLE
Error: local-exec provisioner error

### ISSUE THREAD

**Issue by Hafzan-250601** [ine-labs/AzureGoat#13]
I am getting this error
![image](https://github.com/ine-labs/AzureGoat/assets/54774679/0162e1ef-8f91-4df0-8e30-8d0d5b96b327)

**Comment by RhaXiel** [comment]
TLDR: Use azure cli from the web browser.

For those wondering, there's a video playlist linked in the documentacion: [https://github.com/ine-labs/AzureGoat#solutions](https://github.com/ine-labs/AzureGoat#solutions) 

There's a video tutorial on how to set up the environment and it's actually done directly from azure-cli on the web browser.

My guess is that if you try to terraform on your host, the scripts will use your installed version of python and other dependencies to run all the scripts. At least, I got the same error as you and other 4 different errors when trying to run terraform on different hosts.
